main/main.cpp

```diff
@@ -715,6 +715,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 			editor = true;
 			main_args.push_back(I->get());
+			args.push_back("--no-window");
```

Comment:
It would be better to enable no window mode (it's one line of code) instead of modifying arguments.